### ISSUE TITLE
I want to change the text in Assert.Fail

### ISSUE THREAD

**Issue by shshshshgoodnj** [mattwhitfield/Unitverse#226]
The code I want to fix in Framework Generation.feature
```c# 
        [NUnit.Framework.TestAttribute()]
        [NUnit.Framework.DescriptionAttribute("Framework Generation Test - AssertFail")]
        [NUnit.Framework.TestCaseAttribute("MsTest", "Assert.Fail(\"Create or modify test\");", null)]
        [NUnit.Framework.TestCaseAttribute("NUnit3", "Assert.Fail(\"Create or modify test\");", null)]
        [NUnit.Framework.TestCaseAttribute("XUnit", "throw new NotImplementedException(\"Create or modify test\");", null)]
        public virtual void FrameworkGenerationTest_AssertFail(string framework, string statement, string[] exampleTags)
        {
            string[] tagsOfScenario = exampleTags;
            System.Collections.Specialized.OrderedDictionary argumentsOfScenario = new System.Collections.Specialized.OrderedDictionary();
            argumentsOfScenario.Add("framework", framework);
            argumentsOfScenario.Add("statement", statement);
            TechTalk.SpecFlow.ScenarioInfo scenarioInfo = new TechTalk.SpecFlow.ScenarioInfo("Framework Generation Test - AssertFail", null, tagsOfScenario, argumentsOfScenario, this._featureTags);
``` 

I want to replace the text that is "Asert.Fail(\"Create or modify test\" with "Asert.Fail("Fail") in '[NUnit.Framework.TestCaseAttribute("MsTest", "Assert.Fail(\"Create or modify test\");", null)]'

However, even if I modify the code, it will go back to "Asert.Fail(\"Create or modify test\"
How do I prevent my modified code from being go back even if I build it?

-------------------------------------------------------------------------------------------
This is the result I want when I do the unit test through the code I modified.

Before, Modify
```c#
        [TestMethod]
        public void CanCallGetDataTest()
        {
            // Arrange
            var roiData = _fixture.Create<Info>();

            // Act
            _testClass.GetData(roiData, out var refData, out var refDiffData);

            // Assert
            Assert.Fail("Create or modify test");
        }
``` 

After, Modify
```c#
 [TestMethod]
        public void CanCallGetDataTest()
        {
            // Arrange
            var roiData = _fixture.Create<Info>();

            // Act
            _testClass.GetData(roiData, out var refData, out var refDiffData);

            // Assert
            Assert.Fail("Fail");
        }
```

**Comment by mattwhitfield** [comment]
That's not configurable yet, I'll add something 👍

**Comment by mattwhitfield** [comment]
This should be available in 0.195 💥 Under 'Generation Options' there is 'Default failure message' which you can configure as you please. Please let me know if it works for you

**Comment by shshshshgoodnj** [comment]
I want to modify various Asserts using this principle, so can I know about the algorithm that generates Asserts?
